docs/cpp/customize-cpp-settings-cpp.md

```diff
@@ -1,21 +1,20 @@
 ---
-Order:
+Order: 14
 Area: cpp
-TOCTitle: c_cpp_properties.json
-ContentId: EC1BA944-09B5-41EA-AAED-779A02C90C98
-PageTitle: c_cpp_properties.json reference
-DateApproved: 10/31/2023
-MetaDescription: Schema reference for C++ project settings in Visual Studio Code.
+TOCTitle: Settings Reference
+ContentId: 4E34F6AF-BFC6-4BBB-8464-2E50C85AE826
+PageTitle: Customize settings in Visual Studio Code C++ projects via the c_cpp_properties.json file
+DateApproved: 4/24/2025
+MetaDescription: How to customize the c_cpp_properties.json file for the C++ extension.
 ---
-# c_cpp_properties.json reference
+# C++ extension settings reference
 
-This article explains the scheme for the `c_cpp_properties.json` settings file.
-
-Looking to get started with configuring your project? See [Configure Intellisense](/docs/cpp/configure-intellisense.md). For more information about changing these settings, see [Customizing Default Settings](/docs/cpp/customize-default-settings-cpp.md).
+The C++ Extension settings are highly configurable. This article explains the schema for the `c_cpp_properties.json` file. For general information about settings in VS Code, refer to [User and workspace settings](/docs/configure/settings.md), as well as the [Variables reference](/docs/reference/variables-reference.md) and [Default VS Code Settings](/docs/reference/default-settings.md).
 
+Looking to get started with configuring your C++ project? Begin with [configure Intellisense](/docs/cpp/configure-intellisense.md).
 ## Example of Variables
 
-Note, this is an example of all fields. You do not need to specify all fields in your `c_cpp_properties.json` file. The extension will automatically fill in any missing fields with default values.
+Note, this json is an example of all fields supported in `c_cpp_properties.json`. You don't need to specify all fields in your `c_cpp_properties.json` file. The extension automatically fills in any missing fields with default values.
```

Comment:
Maybe we don't need 3 configs in the example? Can it be made more generic? I'm fine with using linux-style paths (forward slashes) and /usr/bin/g++ as the compilerPath in the remaining config.

Reply:
Originally this was supposed to showcase every single variable we supported, but I am fine with shortening it since most of the variables are the same between different configurations. Are you asking for it to just be one platform's configuration (Ex. only Mac) as an example?

Reply:
If we could shrink to one config, yes, I think macOS would be best since it has the extra property. I was just trying to avoid a JSON overload for the reader.  If you agree, then use clang as the compiler instead of gcc.

Reply:
Ok I updated the sample to make it more generic - let me know if that is what you had in mind

Reply:
Ah I did not see your follow up comment - windows also has the extra property for sdk version, either one could be valuable. I can switch it to Mac with clang++, that is not a big deal

Reply:
Ah, yes I forgot about the windows SDK property. I'll leave it up to you how you want to do it since if we only pick one platform we won't be able to show all the properties. (though I don't think it necessary to show all properties so long as they are documented).